.github/workflows/samples-rust.yaml

```diff
@@ -3,12 +3,14 @@ name: Samples Rust
 on:
   push:
     paths:
-      - 'samples/client/petstore/rust/**'
-      - 'samples/server/petstore/rust-server/**'
+      - "samples/client/other/rust/**"
```

Comment:
i think it should be `others` instead of `other`. Same for below line 11.

Reply:
Ah yes excellent spot. Corrected